src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -57,29 +57,6 @@
    * ./mdx-components/mdx-lists.
    */
 
-	/* inline code elements */
-	& code {
-		background-color: var(--token-color-palette-neutral-50);
-		border-radius: 5px;
-		border: 1px solid rgba(101, 106, 118, 0.2);
-		font-size: 0.8125rem;
-		font-weight: 400;
-		line-height: 16px;
-		padding: 2px 4px;
-	}
-
-	/* code blocks */
-	& pre {
-		border-radius: 5px;
```

Comment:
As far as I can tell, this style wasn't being applied - border radius in `@hashicorp/react-code-block` is applied on a different element, not `pre`. Broader intent here is to handle further refinements to `@hashicorp/react-code-block` in separate PRs.

Reply:
This is probably from the beginning, before we were using CodeBlock 💭